src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -118,30 +118,30 @@ public static function renderProvider()
                 ['ISBN', 'Title', 'Author'],
                 $books,
                 'compact',
-<<<'TABLE'
-ISBN          Title                    Author           
-99921-58-10-7 Divine Comedy            Dante Alighieri  
-9971-5-0210-0 A Tale of Two Cities     Charles Dickens  
-960-425-059-0 The Lord of the Rings    J. R. R. Tolkien 
-80-902734-1-6 And Then There Were None Agatha Christie  
-
-TABLE
+                implode("\n", [
```

Comment:
This change is only because of the line "trim_trailing_whitespace = true" in .editorconfig.